src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerLimits.cs

```diff
@@ -253,13 +253,52 @@ public TimeSpan RequestHeadersTimeout
 
         /// <summary>
         /// Gets or sets the request body minimum data rate in bytes/second.
-        /// Setting this property to null indicates no minimum data rate should be enforced.
-        /// This limit has no effect on upgraded connections which are always unlimited.
-        /// This can be overridden per-request via <see cref="IHttpRequestBodyMinimumDataRateFeature"/>.
+        /// This limit has no effect on upgraded connections.
+        /// This limit can be overridden per-request via <see cref="IHttpMinRequestBodyDataRateFeature"/>.
         /// </summary>
         /// <remarks>
-        /// Defaults to 1 byte/second with a 5 second grace period.
+        /// Defaults to 1 byte/second with a 5 second grace period during which the minimum data rate is not enforced.
         /// </remarks>
-        public MinimumDataRate RequestBodyMinimumDataRate { get; set; } = new MinimumDataRate(rate: 1, gracePeriod: TimeSpan.FromSeconds(5));
+        public MinimumDataRate MinRequestBodyDataRate { get; } = new MinimumDataRate { BytesPerSecond = 1, GracePeriod = TimeSpan.FromSeconds(5) };
+
+        public class MinimumDataRate
```

Comment:
@Tratcher `MinRequestBodyDataRate.BytesPerSecond` matches how the feature is used better than `MinRequestBodyDataRateBytesPerSecond`. It's just aesthetics though.